tools/build_all.py

```diff
@@ -65,8 +65,23 @@ def build_examples(variant):
 
 build_time = time.monotonic()
 
-for var in variants_dict:
-    build_examples(var)
+ENV_VARIABLE_NAME = 'VARIANT'
+
+# build only one variant if the environment variable is specified
+if (ENV_VARIABLE_NAME in os.environ):
+    variant = os.environ.get(ENV_VARIABLE_NAME)
+    # only use the environment variable if the variant exists in the dictionary
+    if (variant in variants_dict):
+        build_examples(variant)
+    else:
+        print('\033[31failed\033[0m - invalid variant name "{}"'.format(variant))
+        fail_count += 1
+        exit_status = -1
+
+else: # no environment variable specified, so build all variants
```

Comment:
nice that you keep the build all option, I usually run this locally before pushing when doing major changes.

Reply:
Yes, I noticed that you checked for travis, and so I thought this might be the case.  👍